test/test-runner/src/runner.rs

```diff
@@ -196,12 +198,20 @@ impl CaseResult {
         let CaseResult { case, result } = self;
         match result {
             Ok(_) => {
+                let mut stdout = StandardStream::stdout(ColorChoice::Always);
```

Comment:
Why not `ColorChoice::Auto`?

Reply:
Because I copy-pasta'd the example from https://docs.rs/termcolor/latest/termcolor/index.html :D